tests/features/testBlueprint.feature

```diff
@@ -4,23 +4,27 @@ Funcionalidade: testBlueprint
   @happy
```

Comment:
É impressão ou vc a cada feature está reescrevendo os testes?
Os testes devem ser incrementais e não sobreescritos.

Reply:
Estou incrementando novos steps nos testes, não reescrevendo.. porém a ordem de alguns steps muda de vez em quando pelo fato de cada nova feature ter um propósito. E dessa vez alterei os inúmeros "Então" consecutivos para a notação "E" do Gherkin, para um entendimento mais fácil na leitura.